pkg/api/v1/client/proxy.go

```diff
@@ -314,6 +314,63 @@ func (c ProxyAPIClient) DeleteExecutor(name string) (err error) {
 	return c.makeDeleteRequest(uri, "testkube", false)
 }
 
+// webhooks --------------------------------------------------------------------------------
+
+func (c ProxyAPIClient) CreateWebhook(options CreateWebhookOptions) (executor testkube.Webhook, err error) {
+	uri := c.getURI("/webhooks")
+
+	request := testkube.WebhookCreateRequest(options)
+
+	body, err := json.Marshal(request)
+	if err != nil {
+		return executor, err
+	}
+
+	req := c.GetProxy("POST").Suffix(uri).Body(body)
+	resp := req.Do(context.Background())
```

Comment:
So the difference with direct is only in above 2 lines, guess, it can be abstracted into some method with options returing response

Reply:
not only this two lines - also getting response is slightly different